README.md

```diff
@@ -22,13 +22,13 @@ let segmenter = new Intl.Segmenter("fr", {granularity: "word"});
 let iterator = segmenter.segment("Ceci n'est pas une pipe");
 
 // Iterate over it!
-for (let {segment, breakType} of iterator) {
-  console.log(`segment: ${segment} breakType: ${breakType}`);
+for (let {segment, precedingSegmentType } of iterator) {
```

Comment:
There’s a space before `}` but no space after `{` after this change.

Reply:
Fixed, thanks.